pkgs/tools/security/kerbrute/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonApplication, fetchPypi, impacket }:
+
+buildPythonApplication rec {
+  pname = "kerbrute";
+  version = "0.0.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-ok/yttRSkCaEdV4aM2670qERjgDBll6Oi3L5TV5YEEA=";
+  };
+
+  # This package does not have any tests
+  doCheck = false;
```

Comment:
Could we have `pythonImportsCheck` in its place at least?

Reply:
(not if there's no importable module obvs.)

Reply:
We should definitely add it if it works.

Reply:
There is no importable module.

Reply:
I would at least do something like

```
installCheckPhase = ''
  $out/bin/kerbrute --version
'';
```

Not a great test, but will signal if the package is trivially working.

Reply:
Fixed it.